1-js/02-first-steps/07-operators/article.md

```diff
@@ -1,119 +1,119 @@
 
-However, note that operations run from left to right. If there are two numbers followed by a string, the numbers will be added before being converted to a string:
+Entretanto, note que as operações são executadas da esquerda para direita. Se há dois números seguidos por uma string, os número serão somados antes e então convertidos para uma string:
 
 
 ```js run
-alert(2 + 2 + '1' ); // "41" and not "221"
+alert(2 + 2 + '1' ); // "41" e não "221"
 ```
 
-String concatenation and conversion is a special feature of the binary plus `+`. Other arithmetic operators work only with numbers and always convert their operands to numbers.
+ Concatenação e conversão de String é uma característica especial do binário mais `+`. Outros operadores aritméticos funcionam apenas com números e sempre convertem seus operandos para números.
 
-For instance, subtraction and division:
+Por exemplo, subtração e divisão:
 
 ```js run
 alert( 2 - '1' ); // 1
 alert( '6' / '2' ); // 3
 ```
 
-## Numeric conversion, unary +
+## Conversão numérica, unário +
 
-The plus `+` exists in two forms: the binary form that we used above and the unary form.
+O mais `+` existe em duas formas: a forma binária que usamos acima e a forma unária.
 
-The unary plus or, in other words, the plus operator `+` applied to a single value, doesn't do anything to numbers. But if the operand is not a number, the unary plus converts it into a number.
+O unário mais ou, em outras palavras, o operador mais `+` aplicado em um único valor, não faz nada para números. Mas se o operando não é um número, o unário mais converte-o em um número.
 
-For example:
+Por exemplo:
 
 ```js run
-// No effect on numbers
+// Sem efeitos em números
 let x = 1;
 alert( +x ); // 1
 
 let y = -2;
 alert( +y ); // -2
 
 *!*
-// Converts non-numbers
+// Converte não números
 alert( +true ); // 1
 alert( +"" );   // 0
 */!*
 ```
 
-It actually does the same thing as `Number(...)`, but is shorter.
+Na realidade faz a mesma coisa que `Number(...)`, porém é mais curto.
 
-The need to convert strings to numbers arises very often. For example, if we are getting values from HTML form fields, they are usually strings.
+A necessidade de converter strings em números surge frequentemente. Por exemplo, se nós estamos pegando valores de campos de formulário HTML, eles são geralmente strings.
 
-What if we want to sum them?
+E se nós quisermos somá-los?
 
-The binary plus would add them as strings:
+O binário mais deve somá-los como strings:
 
 ```js run
 let apples = "2";
 let oranges = "3";
 
-alert( apples + oranges ); // "23", the binary plus concatenates strings
+alert( apples + oranges ); // "23", o binário mais concatena strings
 ```
 
-If we want to treat them as numbers, we need to convert and then sum them:
+E Se nós quisermos tratá-los como números, nós precisamos converter e então somá-los:
 
 ```js run
 let apples = "2";
 let oranges = "3";
 
```

Comment:
- "... nós precisamos **de** converter e então..."

(do **not** include the bold formatting)